deploy-manage/remote-clusters/eck-remote-clusters.md

```diff
@@ -1,26 +1,28 @@
 ---
+applies:
+  eck:
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-remote-clusters.html
 ---
 
-# ECK remote clusters [k8s-remote-clusters]
+# Remote clusters with {{eck}} [k8s-remote-clusters]
 
-The [remote clusters module](https://www.elastic.co/guide/en/elasticsearch/reference/current/remote-clusters.html) in Elasticsearch enables you to establish uni-directional connections to a remote cluster. This functionality is used in cross-cluster replication and cross-cluster search.
+The [remote clusters module](https://www.elastic.co/guide/en/elasticsearch/reference/current/remote-clusters.html) in {{es}} enables you to establish uni-directional connections to a remote cluster. This functionality is used in cross-cluster replication and cross-cluster search.
 
 When using remote cluster connections with ECK, the setup process depends on where the remote cluster is deployed.
 
-## Connect from an Elasticsearch cluster running in the same Kubernetes cluster [k8s-remote-clusters-connect-internal]
+## Connect from an {{es}} cluster running in the same Kubernetes cluster [k8s-remote-clusters-connect-internal]
 
 ::::{note}
 The remote clusters feature requires a valid Enterprise license or Enterprise trial license. Check [the license documentation](../license/manage-your-license-in-eck.md) for more details about managing licenses.
 ::::
 
 
-To create a remote cluster connection to another Elasticsearch cluster deployed within the same Kubernetes cluster, specify the `remoteClusters` attribute in your Elasticsearch spec.
+To create a remote cluster connection to another {{es}} cluster deployed within the same Kubernetes cluster, specify the `remoteClusters` attribute in your {{es}} spec.
 
 ### Security Models [k8s_security_models]
 
-ECK supports two different security models: the API key based security model, and the certificate security model. These two security models are described in the [Remote clusters](https://www.elastic.co/guide/en/elasticsearch/reference/current/remote-clusters.html#remote-clusters-security-models) section of the {{es}} documentation.
+ECK supports two different security models: the API key based security model, and the certificate security model [deprecated in {{es}} 9.0.0]. These two security models are described in the [Remote clusters](https://www.elastic.co/guide/en/elasticsearch/reference/current/remote-clusters.html#remote-clusters-security-models) section of the {{es}} documentation.
```

Comment:
can you remove this reference? perhaps add the definitions that are everywhere else?

Reply:
Yup